manage.js

```diff
@@ -0,0 +1,127 @@
+const express = require('express');
+const app = express();
+const la = require('./luis-add');
+const lr =require('./luis-read');
+const ld =require('./luis-delete');
+const lu =require('./luis-update');
+const da = require('./dialogflow-add');
+const dr =require('./dialogflow-read');
+const dd =require('./dialogflow-delete');
+const du =require('./dialogflow-update');
+const bodyParser = require('body-parser');
+const urlencodedParser = bodyParser.urlencoded({ extended: false });
+app.use(express.static('public'));
+app.use(bodyParser.json());
+app.post('/add', urlencodedParser, function (req, res) {
+    if(req.body.destination==="Microsoft") {
+        la.add(JSON.parse(req.body.utterance))
+            .then((responses) => {
+                console.log(responses);
```

Comment:
You should return the result here, instead of logging it to console. 

It should be `res.send()`, if I remembered it right.